automation/scripts/clean_pipelines/clean_pipelines.py

```diff
@@ -0,0 +1,397 @@
+# Copyright 2019 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+import typing
+from time import sleep
+
+import mlrun_pipelines
+import pandas as pd
+import pytz
+from dateutil import parser
+from kfp import Client
+from mlrun_pipelines.models import PipelineRun
+
+import mlrun
+
+
+def delete_project_old_pipelines(
```

Comment:
All functions in this file can be private except for `delete_project_old_pipelines`